js-exercises/animal-sort/animalSort.js

```diff
@@ -0,0 +1,24 @@
+const animalSort = animals => {
+  if (!(animals instanceof Array)) {
+    throw Error(`Expected Array as an input, got ${typeof animals}`);
+  }
+
+  if (animals.length === 0) {
+    return [];
+  }
+
+  return animals.sort((animalA, animalB) => {
+    if (animalA.numberOfLegs > animalB.numberOfLegs) {
```

Comment:
what if any animal object is missing `numberOfLegs` or `name` or both properties. This program is missing with how it will behave then.
Never believe the input will come as you expect

Reply:
have fixed that in #bcb74da